### PR TITLE
Added Explorer mock checks

### PR DESCRIPTION
**Pull request by skudasov**
Explorer OCR telemetry mock service. Refactored HTTP client for reuse.

### CODE REVIEW

File: environment/templates/explorer-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+spec:
+  selector:
+    matchlabels:
+      app: explorer-mock
+  template:
+    objectmeta:
+      labels:
+        app: explorer-mock
+    spec:
+      containers:
```
Comment (tateexon) [inline environment/templates/explorer-deployment.yml:12]: Do we want to add request/limit values into this spec for the cpu and memory so we avoid adding another unbound deployment?

Comment (skudasov) [reply environment/templates/explorer-deployment.yml:12]: Nice catch, will figure out some limits for it.

Comment (skudasov) [reply environment/templates/explorer-deployment.yml:12]: Currently, we have issues with proper YAML unmarshalling of manifests, think we need to settle them down first.

Comment (skudasov) [reply environment/templates/explorer-deployment.yml:12]: Done

**Review by jleeh — APPROVED** [review]

**Pull request merged**